pom.xml

```diff
@@ -99,6 +99,17 @@
                 <cmake.cflags/>
             </properties>
         </profile>
+        <profile>
+            <id>mac-arm-make</id>
+                <activation>
+                    <property>
+                        <name>build_mac_m1</name>
+                    </property>
+                </activation>
```

Comment:
should there be activation properties set here so that it's the default profile on an M1 mac
```
                <os>
                    <family>mac</family>
                    <arch>arm64</arch>
                </os>
```

otherwise, the existing `mac-make` profile is going to be picked by default, since it specifies `<family>mac</family>`, right?